cli/elastic.py

```diff
@@ -45,3 +47,10 @@ def store_files(index: str, files: [str], cfg_name: str):
                 body=json.load(fh),
                 )
             )
+
```

Comment:
nit: see pycodestyle hints (I just triggered the PR validation job by adding the ok-to-test-label)
-> two blank lines between functions